htslib/1.2.1/Dockerfile

```diff
@@ -0,0 +1,52 @@
+#################################################################
+# Dockerfile
+#
+# Version:          1
+# Software:         htslib
+# Software Version: 1.2.1
+# Description:      C library for high-throughput sequencing data formats
+# Website:          https://github.com/samtools/htslib
+# Provides:         htslib 1.2.1
+# Tags:             Genomics
+# Base Image:       biodckr/biodocker
+# Build Cmd:        docker build --rm -t biodckrdev/htslib 1.2.1/.
+# Pull Cmd:         docker pull biodckrdev/htslib
+# Run Cmd:          docker run --rm -it biodckrdev/htslib <options> <files>
+#################################################################
+
+# Set the base image to Ubuntu
+FROM biodckr/biodocker
```

Comment:
same remarks, should use biocontainers base image and use LABEL for metadata (+ license info)